proto/skaffold.proto

```diff
@@ -376,12 +376,15 @@ enum ClusterType {
 // Enum for Status codes
 // These error codes are prepended by Phase Name e.g.
 // BUILD, DEPLOY, STATUSCHECK, DEVINIT
+// For Success Error codes, use range 200 to 250.
+// For Unknown error codes, use range 500 to 600.
+// For Cancelled Error code, use range 800 to 850.
 enum StatusCode {
     // A default status code for events that do not have an associated phase.
     // Typically seen with the DevEndEvent event on success.
     OK = 0;
 
-    // No Error codes
+    // Success Error codes
```

Comment:
`// Success codes` instead of `// Success Error codes`?

Reply:
done.